docker/README.md

```diff
@@ -103,30 +109,82 @@ You may also want to modify the `-p 80:80` to utilize other ports instead. Refer
 **Note**: if you're using the GPU Docker, skip ahead to the next section.
 
 ```bash
-docker run -it -p 80:80 mms_image:latest
+# Run the docker image in a detached mode
+$ docker run -itd -p 80:80 --name mms -v /home/user/models:/models mms_image:latest
 ```
-Now that you have a prompt inside the container, the final step is to run the following command:
 
+To run the MMS with $HOSTNAME as its endpoint, or for `nginx's server_name` to be configured to `$HOSTNAME` run the following command.
 ```bash
-cd mms_docker_cpu && ./launch.sh
+# Start docker with nginx's server_name configured to $HOSTNAME
+$ docker run -itd -p 80:80 --name mms -v /home/user/models:/models -e MXNET_MODEL_SERVER_HOST=$HOSTNAME mms_image:latest
 ```
+The above command lets you run inference with `$HOSTNAME` as 'server_name'.  
+   
+Verify that this image is running by running 
+```bash
+$ docker ps -a
+CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS                NAMES
+a6141053ef51        mms_cpu             "/bin/bash"         15 minutes ago      Up 15 minutes       0.0.0.0:80->80/tcp   mms
+$
+```
+You should see that the image is running.
+
+## Starting MMS in the docker instance
+To interact with the docker image named `mms` we run the following command
+```bash
+$ docker exec mms bash -c "mxnet-model-server.sh help"
+Usage:
+
+/mxnet_model_server/mxnet-model-server.sh [start | stop | restart | help] [--mms-config <MMS config file>]
+
+start        : Start a new instance of MxNet model server.
+stop         : Stop the current running instance of MxNet model server
+restart      : Restarts all the MMS worker instances.
+help         : Usage help for /mxnet_model_server/mxnet-model-server.sh
+--mms-config : Location pointing to the MxNet model server configuration file.
+To start the MxNet model server, run
```

Comment:
nit: MxNet -> MXNet.
In multiple other places.

Reply:
Done